### PR TITLE
SNOW-716374: reapply telemetry change of application and test panda arrow result format

### PR DESCRIPTION
**Pull request by sfc-gh-aling**
Please answer these questions before submitting your pull requests. Thanks!

1. What GitHub issue is this PR addressing? Make sure that there is an accompanying issue to your PR.

   Fixes #SNOW-716374, pending service change and https://github.com/snowflakedb/snowflake-sqlalchemy/pull/369

2. Fill out the following pre-review checklist:

   - [ ] I am adding a new automated test(s) to verify correctness of my new code
   - [ ] I am adding new logging messages
   - [ ] I am adding new credentials
   - [ ] I am adding a new dependency

3. Please describe how your code solves the related issue.

   Please write a short description of how your code change solves the related issue.

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -3,7 +3,6 @@ universal = 1
 
 [metadata]
 name = snowflake-sqlalchemy
-version = 1.4.6
```
Comment (sfc-gh-sfan) [inline setup.cfg:6]: Should this be updated to 1.4.7 instead?

File: setup.py [hunk 2]
```diff
@@ -2,6 +2,16 @@
...
 #
 
+import os
+
 from setuptools import setup
 
-setup()
+SQLALCHEMY_SRC_DIR = os.path.join("src", "snowflake", "sqlalchemy")
+VERSION = (1, 1, 1, None)  # Default
+with open(os.path.join(SQLALCHEMY_SRC_DIR, "version.py"), encoding="utf-8") as f:
```
Comment (sfc-gh-sfan) [inline setup.py:11]: I could see this being useful, but how is it related to this telemetry change?

**Review by sfc-gh-sfan — APPROVED** [review]

File: setup.cfg (hunk 1 above)
Comment (sfc-gh-aling) [reply setup.cfg:6]: the version number is now in `version.py`

File: setup.py (hunk 2 above)
Comment (sfc-gh-aling) [reply setup.py:11]: we send the version number also part of the telemetry change and version number is used on the backend to check supported versions

Comment (sfc-gh-aling) [reply setup.py:11]: The method that composes telemetry info is already checked in but just not used:
https://github.com/snowflakedb/snowflake-sqlalchemy/blob/main/src/snowflake/sqlalchemy/util.py#L99

**Comment by sfc-gh-aling** [comment]
already checked that telemetry is working in the system.

**Pull request merged**